proto/osmosis/lockup/query.proto

```diff
@@ -95,6 +95,14 @@ service Query {
     option (google.api.http).get =
         "/osmosis/lockup/v1beta1/account_locked_longer_duration/{owner}";
   }
+
+  // Returns account locked records with a specific duration
+  rpc AccountLockedDuration(AccountLockedDurationRequest)
+      returns (AccountLockedDurationResponse) {
+    option (google.api.http).get =
+        "/osmosis/lockup/v1beta1/account_locked_duration/{owner}";
```

Comment:
Since `AccountLockedDurationRequest` defines the query params, is there a point in having `{owner}` as part of the resource URL?

Reply:
Can you elaborate on this? `AccountLockedPastTimeDenom`, `AccountLockedPastTimeNotUnlockingOnlyRequest`, and most of the locked RPC commands have the {owner} within the resource URL, I had assumed it was for the address parameter and maintained practice.

Reply:
@alexanderbez I'm not very familiar with this RPC library - does it make sense for us to create an issue and remove potentially redundant `{owner}`?

Reply:
These urls are mostly used in front end, but we definitely don't need to have them both in the request url and the request itself. Although I don't know if it's worth changing considering the fact that all the urls and queries in the frontend would have to be updated along as well